site/en/_posts/2020-09-22-fair.txt

```diff
@@ -0,0 +1,17 @@
+---
+layout: post
+lang: en
+lang-ref: workshop-fair
+title: "OCR-D at the virtual workshop FAIR & Co"
+tags: [workshop, FAIR, DTA]
+author: Elisabeth Engl
+---
+
+From October 7 to 8, the eHumanities working group of the Union of German Academies of
+Sciences and Humanities, in cooperation with the Academy of Sciences and Humanities in
+Göttingen, is organizing the workshop *FAIR & Co: Visibility and Availability of Digital Academy Research in a Networked Scientific Landscape*.
+The OCR-D project will be represented with a lecture by Matthias Boenig. 
+
+Using the example of DTA, it will be shown how the application spectrum of this reference
```

Comment:
Spelling out what DTA means and what it is might raise interest.

Also, links to the cooperation partners might be handy.

Reply:
good points, done 👍